src/sentry/templates/sentry/layout.html

```diff
@@ -34,14 +33,10 @@
   <title>{% block title %}Sentry{% endblock %}</title>
 
   <script>
-    window.csrfCookieName = '{{ CSRF_COOKIE_NAME }}';
-    window.__sentryGlobalStaticPrefix = '{% asset_url "sentry" "dist/" %}';
     window.__initialData = {% get_react_config %};
-    window.__languageCode = {{ request.LANGUAGE_CODE|to_json|safe }};
   </script>
 
   {% block scripts %}
-  {% include "sentry/includes/sdk-config.html" %}
```

Comment:
If this include is being removed why do you update the file in getsentry?

Reply:
We still use `sdk-config` in getsentry to override the `__initialData.sentryConfig` object. I'm sure I could come up with a better way to specify this configuration for sentry, which we probably will want to come up with a solution for. But for now I'm okay with this solution.